app/parse.py

```diff
@@ -1,9 +1,33 @@
+import datetime
+import csv
 from dataclasses import dataclass
 from urllib.parse import urljoin
 
+from selenium import webdriver
+from selenium.common import (
+    NoSuchElementException,
+    ElementNotInteractableException,
+)
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+driver = webdriver.Chrome()
```

Comment:
The WebDriver is instantiated without any options or configurations. Consider adding options like headless mode or setting a specific path for the WebDriver executable to avoid potential issues.